skins/cat17/src/app/lib/view_handler/section_info.js

```diff
@@ -216,7 +216,7 @@ var objectAssign = require( 'object-assign' ),
 		countryNames: null,
 		update: function ( addressType, salutation, title, firstName, lastName, companyName, street, postcode, city, country, email, aggregateValidity ) {
 			var wrapperTag = '<span>',
-				longtext
+				newLongText
```

Comment:
Took me about 10 seconds to figure out this was a variable declaration and not some text. /me grumbles about how evil this evil JS var declaration of evilness is.